app/data/tasks/fetch_record_csv.py

```diff
@@ -0,0 +1,96 @@
+import csv
+from django.conf import settings
+from django.core.files import File
+from collections import namedtuple
+import tempfile
+
+from celery import shared_task
+from celery.utls.log import get_task_logger
+
+from ashlar.models import Record
+from black_spots.models import BlackspotRecordsFile
+
+
+# Keys which cannot be usefully exported to csv
+DROPPED_KEYS = ['media']
+# static record fields
+RECORD_FIELDS = ['record_id', 'created', 'modified', 'occurred_from',
+                 'occurred_to', 'lat', 'lon', 'location_text',
+                 'city', 'city_district', 'county', 'neighborhood', 'road',
+                 'state', 'weather', 'light']
+
+
+def FIELD_TRANFORMS():
+    def date_iso(d):
+        return d.isoformat()
+
+    FieldTransform = namedtuple('FieldTransform', ['field', 'transform'])
+    return {
+        'record_id': FieldTransform(field='uuid', transform=lambda uuid: str(uuid)),
+        'created': FieldTransform(field='created', transform=date_iso),
+        'modified': FieldTransform(field='modified', transform=date_iso),
+        'occurred_from': FieldTransform(field='occurred_from', transform=date_iso),
+        'occurred_to': FieldTransform(field='occurred_to', transform=date_iso),
+        'lat': FieldTransform(field='geom', transform=lambda geom: geom.y),
+        'lon': FieldTransform(field='geom', transform=lambda geom: geom.x),
+    }
+
+
+@shared_task
+def export_records(occurred_min, occurred_max):
+    records = Record.objects.filter(
+        occurred_from__gte=occurred_min,
+        occurred_to__lte=occurred_max,
+        schema__record_type__label=settings.PRIMARY_RECORD_LABEL
```

Comment:
I think it would be good to keep this function generic by passing in a `record_type_id`; that will make it easy for us to run the script against a different record type if necessary. Right now I don't think that's likely to happen but the general trend on this project has been that a lot of features have ended up being used with multiple record types even when we didn't think they would initially.